ably/realtime/connectionmanager.py

```diff
@@ -426,40 +428,40 @@ def on_suspend_timer_expire():
 
         self.suspend_timer = Timer(Defaults.connection_state_ttl, on_suspend_timer_expire)
 
-    def check_suspend_timer(self, state: ConnectionState):
+    def check_suspend_timer(self, state: ConnectionState) -> None:
         if state not in (
             ConnectionState.CONNECTING,
             ConnectionState.DISCONNECTED,
             ConnectionState.SUSPENDED,
         ):
             self.cancel_suspend_timer()
 
-    def cancel_suspend_timer(self):
+    def cancel_suspend_timer(self) -> None:
         log.debug('ConnectionManager.cancel_suspend_timer()')
         self.__fail_state = ConnectionState.DISCONNECTED
         if self.suspend_timer:
             self.suspend_timer.cancel()
             self.suspend_timer = None
 
-    def start_retry_timer(self, interval: int):
+    def start_retry_timer(self, interval: int) -> None:
         def on_retry_timeout():
             log.info('ConnectionManager retry timer expired, retrying')
             self.retry_timer = None
             self.request_state(ConnectionState.CONNECTING)
 
         self.retry_timer = Timer(interval, on_retry_timeout)
 
-    def cancel_retry_timer(self):
+    def cancel_retry_timer(self) -> None:
         if self.retry_timer:
             self.retry_timer.cancel()
             self.retry_timer = None
 
-    def disconnect_transport(self):
+    def disconnect_transport(self) -> None:
         log.info('ConnectionManager.disconnect_transport()')
         if self.transport:
             self.disconnect_transport_task = asyncio.create_task(self.transport.dispose())
 
-    async def on_auth_updated(self, token_details: TokenDetails):
+    async def on_auth_updated(self, token_details: TokenDetails) -> Optional[asyncio.Future]:
```

Comment:
I think this one should just return `None`, i can see that in some cases it returns a `TokenDetails` but we're not actually using the return value anywhere